tests/test-helper.js

```diff
@@ -20,7 +20,7 @@ Ember.Test.registerHelper('outputPane', function(app) {
 
 Ember.Test.registerHelper('outputContents', function(app, selector) {
   let output = app.testHelpers.outputPane();
-  return output.find(selector).text().trim();
+  return output.document.querySelector(selector).textContent.trim();
```

Comment:
Should `output.document.querySelector(selector)` be checked for truthyness before using `textContent`? I think jquery handled this automatically before. Not sure..

Reply:
If it's not truthy a test will fail. ;)

Reply:
Yeah, but the error won't be pretty haha